index.bs

```diff
@@ -1410,10 +1410,12 @@ authenticator. Since all extensions are optional, this will not cause a function
 Extensions are identified by a string, chosen by the extension author. Extension identifiers should aim to be globally unique,
 e.g., by using reverse domain-name of the defining entity such as `com.example.webauthn.myextension`.
 
+Note: Use of dot-separated notation here does not imply an object hierarchy.
```

Comment:
Sure, added in https://github.com/w3c/webauthn/pull/142/commits/67bf014529ff6d28a52fb32dd373115650dff301